src/library/rdt/src/rdt.c

```diff
@@ -39,6 +39,8 @@ SEXP Rdt(SEXP tracer, SEXP rho, SEXP options) {
         handler = setup_promise_tracing(options);
     } else if (!strcmp("debug", sys)) {
         handler = setup_debug_tracing(options);
+    } else if (!strcmp("specialsxp", sys)) {
```

Comment:
I never said it was not. At that time I was not even super convinced that I should have started top to bottom with a framework :-)

Now it seems that it would be handy.